clang/lib/Headers/opencl-c.h

```diff
@@ -13781,6 +13781,7 @@ float __ovld atomic_fetch_max_explicit(volatile atomic_float *,
     defined(__opencl_c_ext_fp32_local_atomic_min_max)
 
 #if defined(cl_khr_int64_base_atomics) && defined(cl_khr_int64_extended_atomics)
+#ifdef cl_khr_fp64
 #if defined(__opencl_c_ext_fp64_global_atomic_min_max)
```

Comment:
Interesting... maybe `__opencl_c_ext_fp64_global_atomic_min_max` should not be defined if `cl_khr_fp64` is not supported.